model/survey-response-logger.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * @module model/survey-response-logger
+ */
+
+const Sequelize = require('sequelize');
+
+/**
+ * Registers model with Sequelize
+ * @param {Sequelize} sequelize - database instance
+ * @returns {Null} nothing
+ */
+function register (sequelize) {
+    /**
+     * Each SurveyResponseLogger will be associated with a completed SurveyInstance.
+     * Each record represents the timestamps associated with when the question was answered.
+     * It also holds the timestamps for next and previous buttons pressed for each question while filling the survey.
+     * @typedef {Object} SurveyResponseLogger
+     */
+    sequelize.define(
+        'survey_response_logger',
+        {
+            ansTimestamp: {
+                type: Sequelize.DATE
+            },
+            prevTimeStamp: {
+                type: Sequelize.DATE
+            },
+            nextTimeStamp: {
+                type: Sequelize.DATE
+            }
```

Comment:
The latest timestamp of second click on Q1's next button will be logged in for that question and only one timestamp per question for previous and next are saved in the DB